src/interlocking_component/router.py

```diff
@@ -0,0 +1,12 @@
+class Router:
+    """This class will calculate routes for trains from track segment to track segment."""
+
+    def get_route(self, start_track_segment_id: str, end_track_segment_id: str):
```

Comment:
You might want to use `Track`s instead of id's.
(See <https://github.com/BP2022-AP1/bp2022-ap1/pull/66/files#diff-32b59e5349657358f01749afb728eedcd06c4ca2ca854f0b475cc29a31aa8ef5R70>. Yes they currently don't contain a reference to the sumo tracks)